resources/models.py

```diff
@@ -86,7 +86,6 @@ class AbstractArea(TimeStampedModel, UUIDModel):
         help_text=_("ID in the Servicemap system"),
         blank=True,
         null=True,
-        unique=True,
```

Comment:
Why do we have to remove `unique` here?

Reply:
good catch. This change should be in a different PR!